conf/services/feedback.yml

```diff
@@ -1,13 +1,56 @@
 serviceName: Feedback
-serviceHeaderName: Feedback
-serviceDescription: |
-  You can use the feedback survey to tell us how we can make our services better. We might link the feedback
-  you send us to other information about you, like your gender and age. This helps us to better understand how we
-  can improve services.
+serviceHeaderName: ''
```

Comment:
Is this meant to be an empty string?

Reply:
added feedback, dani must have forgot to add it